src/components/app/Home.css

```diff
@@ -8,6 +8,7 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  margin: auto;
```

Comment:
It prevents flex parent container #root default style `align-items: stretch`. Because on small screens .homeSection height more than flex parent container height and with `align-items: stretch` parts that don't fit past to the viewport’s edges, which can’t be scrolled to.

Or other way is to replace `margin: auto` to `align-self: flex-start`.

Reply:
> Or other way is to replace `margin: auto` to `align-self: flex-start`.

I think I prefer this, this feels less "magical".

I thikn you can add `align-self: flex-start` to https://github.com/devtools-html/perf.html/blob/182b796c1405343c83254fc82219bade523c2c78/res/css/style.css#L15-L21

Reply:
No, I can't add `align-self: flex-start` to body, #root and .profileViewer because it breaks profileviewer page. But I'll replace `margin: auto`.

Reply:
Ah right, I didn't try running it with the profile viewer !
When I tried it, I couldn't replace it directly here but I had to apply it at least to the parent (so `#root` ?)